kubernetes-kit-starter/src/main/java/com/vaadin/kubernetes/starter/sessiontracker/serialization/debug/SerializationDebugRequestHandler.java

```diff
@@ -233,13 +247,26 @@ public static void serializeAndDeserialize(WrappedSession session,
         }
     }
 
+    private int getSerializationTimeout(SerializationProperties properties) {
+        int timeout = DEFAULT_SERIALIZATION_TIMEOUT_MS;
+        if (properties != null && properties.getTimeout() > 0) {
+            timeout = properties.getTimeout();
+        } else {
+            String timeoutStr = System.getProperty(SERIALIZATION_TIMEOUT_PROPERTY);
```

Comment:
We could use `ApplicationConfiguration.getStringProperty()` to be consistent with other components getting Flow configuration values.

Reply:
Do you mean that we could get the system property through that?

Reply:
Yes, `ApplicationConfiguration` it has multiple fallbacks: system property and application properties (e.g. servlet init paramters). It is widely used in Flow to get application properties

Reply:
But it probably expects the property name without the `vaadin.` prefix. I have to check

Reply:
Yes, implementation appends the `vaadin.` prefix. 
So, keep it simple and use `System.getProperty()` as it was since the beginning